idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -172,6 +176,7 @@ export default PageView.extend({
     app.on('webChannel:addon-install:install-ended', () => {
       this.model.enabled = !this.model.enabled;
       evt.target.classList.remove('state-change');
+      this.model.fetch();
```

Comment:
It'd be good to manually bump the installation_count in the view before firing the xhr, so that the install button and the counter appear to update at the same time.